public/less/_repository.less

```diff
@@ -899,6 +899,10 @@
                 margin: 15px 0 15px 79px;
                 padding-left: 25px;
 
+                & > .octicon:not(.issue-symbol) {
+                    text-shadow: -2px 0 #ffffff, 0 2px #ffffff, 2px 0 #ffffff, 0 -2px #ffffff;
```

Comment:
Could use shorthand colors like `#fff`.

Reply:
strange:
- i know this is possible in css
- but css linter (make css) always said i should change this to full length
- if i change this to short form in .less, currently there is no warning...

-> in rendered css there is already the short form

Reply:
> but css linter (make css) always said i should change this to full length

Check if your linter actually uses stylelint or something else. We use stylelint and explicitely disable the [color-hex-length](https://github.com/go-gitea/gitea/blob/49ee9d27718c31d5ab4fe9543fefc90cd40d4405/.stylelintrc#L5) rule so the linter has no preference currently (but I want to eventually enable it to the short form).

> in rendered css there is already the short form

That's the minifier (clean-css module) optimizing for size.

Reply:
i've changed to short form